src/code/sourceLocationTracker.js

```diff
@@ -67,7 +67,7 @@ function getSourceMap (address, code, contracts) {
   var isCreation = helper.isContractCreation(address)
   var byteProp = isCreation ? 'bytecode' : 'runtimeBytecode'
   for (var k in contracts) {
-    if ('0x' + contracts[k][byteProp] === code) {
```

Comment:
hm.. at this point of the code, we are a bit far far away from the parameters declaration.
Actually we don't access the input parameters at all in remix I think